src/nimblepkg/forge_aliases.nim

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) Dominik Picheta. All rights reserved.
+# BSD License. Look at license.txt for more info.
+
+import std/[strutils]
+import common, download
+
+type
+  ForgeKind* = enum
+    fgGitHub
+    fgGitLab
+    fgSourceHut
+    fgCodeberg
+
+  Forge* = ref object
```

Comment:
Why is that a `ref`? It doesn't look like it needs reference semantics.

Reply:
Ah, I'd assumed that objects with field discriminators need to be `ref`s.